src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -567,7 +567,7 @@ public partial interface ISecurityPolicyEncodable
         void FromXml(System.Security.SecurityElement e, System.Security.Policy.PolicyLevel level);
         System.Security.SecurityElement ToXml(System.Security.Policy.PolicyLevel level);
     }
-#if !netcoreapp && !uap
+#if !netcoreapp && !netcoreapp30 && !uap
```

Comment:
actually those ifdefs imply that this will only build on netfx. I think switching the condition to #if netfx would make more sense.

Reply:
They build on netfx and netstandard. I can change it to #if netstandard || netfx

Reply:
I don't have a strong opinion here, whatever you think makes more sense.